doc/source/user_guide.md

```diff
@@ -513,3 +516,73 @@ sensitive_unshuffled_avatars_df = client.pandas_integration.download_dataframe(
 )
 print(sensitive_unshuffled_avatars_df.head())
 ```
+
+## How to launch an avatarization with time series
+
+An avatarization job can be launched on data containing time series. For more details on time series avatarization, see the [dedicated notebook tutorial on time series]((https://github.com/octopize/avatar-python/blob/main/notebooks/Tutorial8-Time_series.ipynb)). 
+
```

Comment:
this is really good to link our tutorial into the documentation, but IMO, we shouldn't put code in the `How to`

Reply:
could you add a link to the public doc?

Reply:
I would disagree. I believe the how to provides quick answer and solutions to the main questions a user can have. So how to run a basic anonymization is a question that is worth being listed here. We have the public doc for the concept part and the tutorial for the complete code example with explanations, the user guide comes in between. 
We could discuss the role of the 3 sources of docs as a team.